src/stories/text/Text.tsx

```diff
@@ -1,22 +1,38 @@
 import type { ComponentProps, FC, ReactNode } from 'react';
 import { createElement } from 'react';
 import clsx from 'clsx';
-import styles from './Text.module.scss';
 import typography from './Typography.module.css';
+import styles from './Text.module.scss';
 import type { LightTheme } from '../theme';
 
 export type TextElement = 'p' | 'span' | 'div' | 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'label' | 'legend' | 'caption' | 'small';
+export type GlobalCSSValues = 'inherit' | 'initial' | 'revert' | 'revert-layer' | 'unset';
+export type FontWeight = 100 | 200 | 300 | 400 | 500 | 600 | 700 | 800 | 900 | 'normal' | 'bold' | 'lighter' | 'bolder' | GlobalCSSValues;
```

Comment:
technically per [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#values), any number between 1-1000 should be valid for font weight:

> A [<number>](https://developer.mozilla.org/en-US/docs/Web/CSS/number) value between 1 and 1000, inclusive.

similarly an [`oblique <angle>`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style#variable_fonts) modifier should be valid for fontStyle. these are both important to ensure support for variable fonts (e.g. Inter var).

for both `FontWeight` and `FontStyle` (and any typed CSS properties in general) we should always either:

* use `csstype`'s provided type for the property
* if we _need_ stricter typing, add to our [`@ssh/csstypes`](https://www.npmjs.com/package/@ssh/csstypes) library (though in general this approach should be limited to opinionated theme variables that can easily be overridden rather than in component props)

in this case, though, I believe we should actually define our own semantic weight and style variables in the main Paris theme (e.g. `extralight, light, normal, medium, semibold, bold, extrabold` mapped to whatever weights we want available per our preferred typeface or design system) which are then provided as options for these props, rather than making them seem like a direct CSS value passthrough

imo this approach is a lot less confusing, since the consumer is expected to just pass an inline style or custom class (which thus supports CSS modules and CSS-in-JS) whenever they need to directly override/customize CSS values. styling props in any component we build should be used only for predefined customizations per our style system